test/everypolitician/popolo/event_test.rb

```diff
@@ -18,13 +18,21 @@ def test_no_events_in_popolo_data
 
   def test_accessing_event_properties
     popolo = Everypolitician::Popolo::JSON.new(
-      events: [{ id: 'term/8', name: '8th Verkhovna Rada', start_date: '2014-11-27' }]
+      events: [{ id:              'term/8',
+                 start_date:      '2014-11-27',
+                 classification:  'legislative period',
+                 name:            '8th Verkhovna Rada',
+                 organization_id: '2fd16480-aa16-4055-bcad-66bfaae6f18b',
+                 wikidata:        'Q123456', },]
```

Comment:
Have you seen a Term actually being defined like this within EveryPolitician data? This doesn't seem to match how that term is currently set up in `master`, or how I would expect that to look. I think you should be testing based on a real ep-popolo-v1.0.json file.